js/jsmind.draggable-node.js

```diff
@@ -274,6 +278,32 @@
                 var jview = this.jm.view;
                 var px = (e.clientX || e.touches[0].clientX) / jview.actualZoom - this.offset_x;
                 var py = (e.clientY || e.touches[0].clientY) / jview.actualZoom - this.offset_y;
+                // scrolling container axisY if drag nodes exceeding container
+                if (
+                    e.clientY - this.innerElSize.top < 10 &&
```

Comment:
we'd better put `10`, `15`, etc to the `options` on the top of this file, so we can customize them.

Reply:
that's good idea,but this value was experimentally determined by me, so customization may cause problems in different scenarios

Reply:
For me, `10` is too small to be triggered , I prefer to increase it to `20` or `30`. So I think an option of it would be more friendly.